packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -320,6 +320,43 @@ class AndroidSdk {
 
   String? getAvdManagerPath() => getCmdlineToolsPath(globals.platform.isWindows ? 'avdmanager.bat' : 'avdmanager');
 
+  String? getNdkCompilerPath(String binaryName) {
+    final Directory ndk = directory.childDirectory('ndk');
+    if (!ndk.existsSync()) {
+      return null;
+    }
+    final List<Directory> ndkVersions =
+        ndk.listSync().whereType<Directory>().toList()..reversed;
```

Comment:
According to the docs [listSync](https://api.flutter.dev/flutter/dart-io/Directory/listSync.html) returns the entries in an arbitrary order so why the reversed call?